src/context/AirDBContext.tsx

```diff
@@ -71,6 +81,129 @@ export const AirDBProvider = React.memo((props: any) => {
     )
   }
 
+  const handleLogin = ({ password, username, setToggled }: any) => {
+    const isMatching =
+      users.filter(
+        (user: { fields: { password: string; username: string } }) =>
+          user.fields.password === password && user.fields.username === username
+      ).length > 0
+    const userId = (
+      users.filter(
+        (user: { fields: { username: string } }) =>
+          user.fields.username === username
+      )[0] || {
+        id: ''
+      }
+    ).id
+
+    if (isMatching) {
+      setToggled(false)
+      window.localStorage.setItem('username', username)
+      return updateAirDB('Testy - Users', userId, {
+        active: 'true'
+      })
+    } else {
+      throw new Error('INCORRECT PASSWORD')
```

Comment:
Does this error get caught and presented to the user?

Reply:
No, I will work on that later.